### PR TITLE
I have added new file

### PR DESCRIPTION
**Pull request by chiragkhimani**
Please approve and coffee on me.

### CODE REVIEW

**Comment by vildanembel** [comment]
nice work

**Comment by vildanembel** [comment]
approved

**Review by chiragkhimani — COMMENTED** [review]
Looks good

File: src/test/java/com/automation/test/DemoTest.java [hunk 1]
```diff
@@ -7,7 +7,5 @@ public static void main(String[] args) {
 		System.out.println("thir line from chirag");
 		System.out.println("second line ");
 		System.out.println("third line from Guljan");
-
```
Comment (chiragkhimani) [inline src/test/java/com/automation/test/DemoTest.java:10]: This is nice work!

**Pull request merged**